.github/workflows/main.yml

```diff
@@ -8,7 +8,8 @@ jobs:
     strategy:
       matrix:
         # os: [ubuntu-18.04, windows-2019, macos-10.15]
-        os: [ubuntu-18.04]
+        os: [ubuntu-18.04, windows-2019]

```

Comment:
with this, we could drop appveyor, in a next MR we could also add mac os, and execute tests here...